.circleci/config.yml

```diff
@@ -0,0 +1,86 @@
+version: 2
+
+jobs:
+  build:
+    environment:
+      HUGO_VERSION: 0.31.1
+      HUGO_THEME: https://github.com/Sabayon/sabayon-website-next-theme
+
+    docker:
+      # Since there are no official Hugo docker builds, pick one from the
+      # community.
+      # See https://discourse.gohugo.io/t/docker-official-repo-for-hugo/144/6
+      - image: jojomi/hugo:0.31.1
```

Comment:
Since we are not using hugo from the image, i guess it's fine to use ours in this case